doc/changes/changes_15.3.0.md

```diff
@@ -0,0 +1,9 @@
+# Common module of Exasol Virtual Schemas Adapters 15.3.0, released 2021-??-??
```

Comment:
You don't want to release it now, right?

Reply:
Not yet, It will be also tested in the core team